examples/wave_example.ipynb

```diff
@@ -25,154 +25,1751 @@
+       "2018-01-01 04:40:00     0.0     0.0     0.0     0.0     0.0    0.03    0.14   \n",
+       "\n",
+       "                     0.0625  0.0675  0.0725  ...  0.3300  0.3400  0.3500  \\\n",
+       "2018-01-01 00:40:00    0.09    0.22    0.22  ...    0.07    0.08    0.07   \n",
+       "2018-01-01 01:40:00    0.08    0.21    0.21  ...    0.07    0.07    0.08   \n",
+       "2018-01-01 02:40:00    0.07    0.14    0.32  ...    0.08    0.06    0.05   \n",
+       "2018-01-01 03:40:00    0.22    0.26    0.32  ...    0.05    0.06    0.05   \n",
+       "2018-01-01 04:40:00    0.17    0.37    0.40  ...    0.06    0.12    0.04   \n",
+       "\n",
+       "                     0.3650  0.3850  0.4050  0.4250  0.4450  0.4650  0.4850  \n",
+       "2018-01-01 00:40:00    0.07    0.03    0.01    0.02    0.01    0.01     0.0  \n",
+       "2018-01-01 01:40:00    0.02    0.01    0.02    0.01    0.01    0.00     0.0  \n",
+       "2018-01-01 02:40:00    0.03    0.03    0.02    0.01    0.01    0.00     0.0  \n",
+       "2018-01-01 03:40:00    0.03    0.03    0.01    0.01    0.00    0.00     0.0  \n",
+       "2018-01-01 04:40:00    0.04    0.02    0.01    0.01    0.01    0.01     0.0  \n",
+       "\n",
+       "[5 rows x 47 columns]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# Set the random seed, to reproduce results\n",
-    "np.random.seed(1) \n",
+    "ndbc_data_file = '../mhkit/tests/data/data.txt'\n",
+    "\n",
+    "[raw_ndbc_data, meta] = wave.io.read_NDBC_file(ndbc_data_file) # read_NDBC_file outputs the NDBC file data into two variables.\n",
+    "                                            # data is a pandas DataFrame containing the file data. \n",
+    "                                            # meta contains the meta data, if available. \n",
     "\n",
-    "Hm0 = pd.Series(np.random.rayleigh(2, 1000000)) # significant wave height values\n",
-    "Te = pd.Series(np.random.normal(4.5, .7, 1000000)) # peak period values\n",
-    "P = pd.Series(np.random.normal(200, 40, 1000000)) # power values\n",
-    "J = pd.Series(np.random.normal(300, 8, 1000000)) # omnidirectional wave flux values"
+    "raw_ndbc_data.head()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Capture Length Matrices\n",
-    "\n",
-    "The following operations create capture length matrices, as specified by the IEC/TS 62600-100.  But first, we need to calculate capture length and define bin centers.  The mean capture length matrix is printed below.  Keep in mind that this data has been artificially generated, so it may not be representative of what a real-world scatter diagram would look like."
+    "The resulting data DataFrame is spectra indexed by datetime with frequency as the columns. To use this data in MHKiT functions we must first transpose the DataFrame"
```

Comment:
I cannot find a reference for this in the documentation (e.g. where is it specified that DataFrames must be indexed by frequency?).

Reply:
@ssolson it's here https://mhkit-software.github.io/MHKiT/mhkit-python/api.wave.html

Reply:
Okay I was going to add a reference to it. Once the Capture length matrix calculation is resolved we can address this.

Reply:
We may want to make it more prominent, but for now it's in the source code itself, and it's also there as a note on the required spectral data format.